src/mccann_hub/odoolib/connector/__init__.py

```
@@ -0,0 +1,4 @@
+from .json_rpc import JsonRpcConnector, JsonRpcException
```

Ruff (0.8.2)

`.json_rpc.JsonRpcConnector` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)